WalkingChallenge/AppController.swift

```diff
@@ -81,7 +85,11 @@ class AppController {
   }
 
   func login() {
-    transition(to: .navigation)
+    if !Settings.OnboardingComplete {
```

Comment:
What style guide are we following? It wasn't clear to me that ```onboardingComplete``` was a boolean since it's capitalized

Reply:
Sorry, I tend to write a ton of LLVM code which uses StudlyCaps, so its a force of habit.  I can change that to `onboardingComplete`.